modules/apps/object/object-api/src/main/java/com/liferay/object/entry/util/ObjectEntryDTOConverterUtil.java

```diff
@@ -74,6 +77,35 @@ public static Object toDTO(
 		return dtoConverter.toDTO(defaultDTOConverterContext);
 	}
 
+	public static String toDTO(
+			DTOConverterRegistry dtoConverterRegistry, JSONFactory jsonFactory,
+			ObjectEntry objectEntry, User user)
+		throws Exception {
+
+		DTOConverter<ObjectEntry, ?> dtoConverter =
+			(DTOConverter<ObjectEntry, ?>)dtoConverterRegistry.getDTOConverter(
+				ObjectEntry.class.getName());
+
+		DefaultDTOConverterContext defaultDTOConverterContext =
+			new DefaultDTOConverterContext(
+				false, null, dtoConverterRegistry, null, user.getLocale(), null,
+				user);
+
+		Object object = dtoConverter.toDTO(
+			defaultDTOConverterContext, objectEntry);
```

Comment:
I would rename it to `dto`

Reply:
Done, I used this name because there is no conflict names, but I also prefer to use a clear name.